### ISSUE TITLE
Improve namespace-filter expansion UX.

### ISSUE THREAD

**Issue by foot** [weaveworks/scope#3555]
https://github.com/weaveworks/scope/pull/3117 made having a lot of namespaces more manageable in the UI.

There were some ideas to improve this even further:
- @AVRahul:
  > there are some improvements which could be better, the hovering effect can be just restricted to the last-child class instead applying in overall sidebar.
  
- Provide an explicit "show more" button to expand and collapse the namespace list.

**Comment by dholbach** [comment]
@AVRahul: did you have a different solution than #3117 in MayaOnline? Or improvements to it?

**Comment by AVRahul** [comment]
sorry for the delay, @dholbach  as per the current scenerio in MayaOnline we have an improvement as mentioned above. We have restricted the hover effect just to last-child css class to maintain better experience in all the browsers. Because when we checked it in firefox, srollbar appears even when there are less namespaces as shown below. So we fixed this issue.  

![screenshot](https://user-images.githubusercontent.com/10995416/51356631-48a13900-1ae1-11e9-8aa7-d24c4f96e5a7.png)
